src/components/NavBar.jsx

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import { useState } from 'react';
+import BackcountryLogo from '../logos/backcountry-logo-with-text.svg';
+import NavBarSwitchingIcon from '../logos/NavBarSwitchingIcon.svg';
+import { useNavigate, useLocation } from 'react-router-dom';
+import useSessionStorage from "../hooks/useSessionStorage";
+
+function NavBar() {
+    const navigate = useNavigate();
+    const location = useLocation();
+    const [showPopup, setShowpopup] = useState(false)
+    const [name] = useSessionStorage("userName");
+    const handlPopup=()=>{
+        setShowpopup(!showPopup)
+    }
+    
+    return (
+    <nav className="bg-white mt-2 mb-2">
+    <div className="flex flex-wrap justify-between items-center mx-auto max-w-screen-xl">
+    <div className="flex items-center ml-15">
+    <img 
+       className="ml-8 px-1 py-1 h-26 w-147"
+       src={BackcountryLogo} alt="Backcountry SVG"/>
+    </div>
+        <div className="flex items-center mr-10">
+           {location.pathname !== '/menuChooser' && (<div className='mr-2'>
+                <button data-testid="manualflow-button" onClick={()=> navigate('/manualWorkFlowDashboard')}> 
+                <div className="bg-black text-white text-sm rounded-full border h-8 w-8 px-1 py-1"> + </div>  
+            </button>
+            </div>)}
+       
+            {location.pathname !== '/menuChooser' && (<div className='mr-2'>
+             <button> 
+                <img
+                  className="px-1 py-1 h-31 w-31"
+                  src={NavBarSwitchingIcon}
+                  alt="NavBarSwitchingIcon SVG"/>   
+             </button>
+           </div>)}
+
+           <div className='mr-2'>
+            <button onClick={handlPopup}> 
```

Comment:
Make button as a reusable component

Reply:
This button need not to be a separate component as we are changing the classes of the button as per the requirement.

Reply:
Then pass classes as a prop to the button component.

Reply:
Button has to be a separate component.